### DISCUSSION TITLE
Cef build defines 91.1.211

### DISCUSSION THREAD

**Discussion by cyberfizz** [cefsharp/CefSharp#3664] [Q&A]
Hi,

What is the criteria needed for CEF when building manually and moving the files across as defined in the output file description table?

I manually built CEF and used branch 4772, copying across the files from both the Debug_GN_x86 folder and Debug_GN_x64 (made using 'ninja -C out\Debug_GN_x86 cef' commands) but i ended up getting a file can't load error on cef.core.runtime. Is there a specific build criteria like  set GN_DEFINES=is_component_build=true? The cefclient application works ok, so suspect its a mismatch on either version or build type.

**Comment by amaitland** [comment]
> Cef build defines 91.1.211

What do you mean by this? 

> What is the criteria needed for CEF when building manually and moving the files across as defined in the output file description table?

You need to build the exact commit that was used otherwise you'll need to rebuild the libcef wrapper. 

The commit is included in the CEF version number which you can get from the release notes or opening chrome://version in the browser. 

> Is there a specific build criteria like set GN_DEFINES=is_component_build=true? 

is_component_build is for local development and not distribution. You need to create a official build as defined at https://bitbucket.org/chromiumembedded/cef/wiki/AutomatedBuildSetup.md#markdown-header-windows-configuration

**Reply by cyberfizz** [reply]
Thanks Alex, i will change it to use the is_official_build=true instead and regrab the sources. Weirdly the version is exactly the same:
91.1.21+g9dd45fe+chromium-91.0.4472.114

would it be because its component build then and not official build? Or does it make a difference if the ninja compile is Debug vs Release?

Sorry new to building things from source and really trying to be self sufficient.... not doing a great job.

**Reply by amaitland** [reply]
You need a official release build. Soon non component debug builds will be blocked https://bitbucket.org/chromiumembedded/cef/commits/de46befc1821ebe65e332cc28bebda2c4c792f87
